examples/cli/readme.md

```diff
@@ -1 +1,619 @@
-placeholder
\ No newline at end of file
+# Learning the `benchit` CLI
+
+This document is a tutorial for exploring the different features of the `benchit` command line interface (CLI). You can learn the details of those features in the [Tools User Guide](https://github.com/groq/mlagility/blob/main/docs/tools_user_guide.md) and learn about their implementation in the [Code Organization Guide](https://github.com/groq/mlagility/blob/main/docs/code.md).
+
+We've created this tutorial document because `benchit` is a CLI that benchmarks the contents of `.py` scripts. So all of the `.py` scripts in the `examples/cli/scripts` directory are meant to be fed into `benchit` to demonstrate some specific functionality.
+
+Once you've familiarized yourself with these features, head over to the [`models` directory](https://github.com/groq/mlagility/tree/main/models) to learn how to use `benchit` with real world machine learning models.
```

Comment:
You will not really "learn" anything by going to a folder full of models. Maybe you can "start using" instead of "learn how to use"?

Reply:
When you click the link you will see https://github.com/groq/mlagility/tree/main/models/readme.md because that's how github is rendered. Are you reviewing based on the markdown sourcecode or the way the markdown is rendered and used?

Reply:
I see. That sounds good then.